lexic.el

```diff
@@ -652,7 +652,7 @@ Returns a list of plists with keys :word, :dict, and :info."
   "When lexic failed to match the word, format the suggestions in RESULTS."
   (let (suggestions last-match)
     (while (setq last-match
-                 (string-match "^[0-9]+)\\(.*\\)-->\\([A-Za-z]+\\)"
+                 (string-match "^[0-9]+)\\(.*\\)-->\\(.*\\)"
```

Comment:
This makes sure to include the whole suggestion, so that it can be searched.
For example when searching for "project" in the "Online Etymology Dictionary", it returns two suggestions:
- project (n.)
- project (v.)
allowing only alpha characters would not work